handler_test.go

```diff
@@ -121,13 +99,14 @@ func TestHandler_DeleteDatabase_NotFound(t *testing.T) {
 }
 
 func TestHandler_RetentionPolicies(t *testing.T) {
+	t.Skip()
```

Comment:
They sure are. Since we didn't have the necessary influxql commands, the tests couldn't run. We do now, so I'll be re-enabling.